src/raytracer/clusterPaa.m

```diff
@@ -136,80 +136,83 @@
             % it.
 
             % Update paaInfo struct
-            paaInfo{node_id}.paaInCluster{idx_paa} = paaCluster;
-            paaInfo{node_id}.centroids(idx_paa) = paaFullCorr(popularPaaId); % Store centroid label
-            paaTimePosition(:,idx_paa, :) =squeezeAndReshape(nodeTimePosition(:,node_id, :)) + ...
-                repmat(nodePaaPosition{node_id}(paaInfo{node_id}.centroids(idx_paa),:), ...
+            paaInfo{nodeId}.paaInCluster{idxPaa} = paaCluster;
+            paaInfo{nodeId}.centroids(idxPaa) = paaFullCorr(popularPaaId); % Store centroid label
+            paaTimePosition(:,idxPaa, :) =squeezeAndReshape(nodeTimePosition(:,nodeId, :)) + ...
+                repmat(nodePaaPosition{nodeId}(paaInfo{nodeId}.centroids(idxPaa),:), ...
                 [numberTimeDivision, 1,1]);%#ok<*EMGRO> % Add PAA shifts to PAA locations to compute PAA position in the global frame
-            paaInfo{node_id}.generationMethod(idx_paa) = 0;
+            paaInfo{nodeId}.generationMethod(idxPaa) = 0;
             paaClustered = [paaClustered; paaCluster];  %#ok<AGROW>
-            paaInfo{node_id}.centroidsShift{idx_paa} = repmat(nodePaaPosition{node_id}(paaInfo{node_id}.centroids(idx_paa),:), ...
+            paaInfo{nodeId}.centroidsShift{idxPaa} = repmat(-nodePaaPosition{nodeId}(paaInfo{nodeId}.centroids(idxPaa),:), ...
                 length(paaCluster),1)...
-                -nodePaaPosition{node_id}(paaCluster ,:); % Save shift from centroid
-            paaInfo{node_id}.orientation{idx_paa} = nodePaaOrientation{node_id}(paaCluster, :);
+                +nodePaaPosition{nodeId}(paaCluster ,:); % Save shift from centroid
```

Comment:
are these changes of sign ok? just checking that were not changed by mistake, i honestly have no clue